### ISSUE TITLE
Build is not working in the it-ad-wip-italic-v1.002

### ISSUE THREAD

**Issue by MariannaPaszkowska** [googlefonts/googlesans-flex#532]
https://github.com/googlefonts/googlesans-flex/actions/runs/5121854566/jobs/9210140961#step:7:222

**Comment by RickyDaMa** [comment]
The issue looks to be caused by malformed guidelines in both `fontinfo.plist` as well as some individual `glif` files. I can see that these issues have been present since the inception of the `it-ad-wipitalic-v1.002`, as there has yet to be a successful build

According to @madig, guidelines rely on a UFO specification update that isn't yet fully supported by `ufolib2` and `fonttools.ufolib`. While this is in progress (I believe Nikolaus wants to or is working on the former), for now he's suggested we 'ban' guidelines from sources to get everything building ASAP. Nikolaus will write a script to remove guidelines from the sources, use it to clean the branch, which Octavio can then pull and work off of, taking care not to add any guidelines back in

Final note from me is that due to this, I'm currently blocked on testing italics import. As I said previously in the meeting, I believe it should 'just work' with no additional changes necessary, but I'm not able to validate it against this branch until it can be built

**Comment by madig** [comment]
The source `sources/italic/ROND100/GoogleSansFlexItalic-opsz6-wdth100-wght1000-ROND100.ufo` is malformed and cannot be loaded, so I can't proceed until I know why it's broken. It seems to have been mostly deleted in commit b07f9525a1f045863feb38c8824767f381cf28fa, but leftovers remain and break compilation.

**Comment by MariannaPaszkowska** [comment]
@octaviopardo it seems that the guidelines are causing the build to break. Can we commit UFO files without the guidelines?

**Comment by MariannaPaszkowska** [comment]
@madig, thanks for fixing the guidelines; we have a new issues: 

"ERROR:ufo2ft.instructionCompiler:The stored hash for glyph 'Euro' does not match the TrueType output glyph. Glyph will have no instructions in the font."

https://github.com/googlefonts/googlesans-flex/actions/runs/5141525179/jobs/9254118129#step:7:2388 

CC @octaviopardo any idea why this is happening?

**Comment by octaviopardo** [comment]
Yes, we are working on it. It is related with the features file. I am hoping to solve it in the next hour

**Comment by madig** [comment]
It's not the feature file, it's the empty TrueType assembler blocks in the regular sources. Georg's test branch dropped them, I can drop them as well on the branch. You can ignore the errors fortunately.

**Comment by MariannaPaszkowska** [comment]
@madig found the bug: 
"ringcomb: sources/italic/GoogleSansFlexItalic-opsz144-wdth25-wght400-ROND0.ufo/glyphs/ringcomb.glif
has an advance width of 1, but it should be zero. That makes things build."

CC @octaviopardo

**Comment by joanacorreiatype** [comment]
Corrected and commited as Update[GSI-opsz144-wdth25-wght400-ROND0-Milestone02]. (commit code: 8abe9050e12b508d98355cfe6c55323b9d6aa6de)

**Comment by MariannaPaszkowska** [comment]
Another issue in the sources @octaviopardo @joanacorreiatype 
"fontmake.errors.FontmakeError: In 'italic/GoogleSansFlex-Italic.designspace': Generating fonts from Designspace failed: The width should not be negative: 'tildecomb'
make: *** [Makefile:23: build.stamp] Error 1"

**Comment by Txonliz** [comment]
Corrected side bearings in "tildecomb" [GoogleSansFlexItalic-opsz18-wdth25-wght1000-ROND100].
Width was 0 (see screenshot):
 
![Screenshot_2023-06-02](https://github.com/googlefonts/googlesans-flex/assets/10536858/110e5b96-b0cd-427d-a363-88156f7e8511)

**Comment by MariannaPaszkowska** [comment]
It seems to still be an issue: https://github.com/googlefonts/googlesans-flex/actions/runs/5154934235/jobs/9283971785#step:7:212

**Comment by joanacorreiatype** [comment]
I checked the file, and there is no issue with the tildcomb. There is no negative width. 

<img width="974" alt="Screenshot 2023-06-02 at 12 40 48" src="https://github.com/googlefonts/googlesans-flex/assets/9676919/f816149f-39af-43c4-883b-130558546cb5">

**Comment by octaviopardo** [comment]
I ll make @casasin look at this, but maybe @madig can also look at it to make sure that it s not something wrong on their side?

**Comment by madig** [comment]
It's -11 in GoogleSansFlexItalic-opsz144-wdth100-wght400-ROND0.ufo.

**Comment by octaviopardo** [comment]
We are not sure if this is the main reason but we are detecting issues of consistency in many UFOs in between the regular in our branch and our italic sources. We've double checked and the anchors in the upright sources of the main branch and the sources in our branch seem to be different as well. How can we address this issue?

**Comment by casasin** [comment]
The problem seems to be in anchors consistency across masters in several files (I'd say not only in "GoogleSansFlexItalic-opsz144-wdth100-wght400-ROND0.ufo"). Did the sources in our branch build before we touched any file?

**Comment by madig** [comment]
There's a recurring problem, too: sources/italic/ROND100/GoogleSansFlexItalic-opsz6-wdth100-wght1000-ROND100.ufo is back for the third time and breaking the build as before. Can you please check all of your repo copies for this, including the Designspace files, and _remove it_? It was last added back in by Joancarles Casasín in commit 25f24c57ac6b724af37e3db24469689aef52fc49, but it might be lurking elsewhere, too.

**Comment by madig** [comment]
The other build breaker is a "B" with an open contour in sources/italic/ROND100/GoogleSansFlexItalic-opsz18-wdth100-wght400-ROND100.ufo.

**Comment by casasin** [comment]
> The other build breaker is a "B" with an open contour in sources/italic/ROND100/GoogleSansFlexItalic-opsz18-wdth100-wght400-ROND100.ufo.

This has been fixed right now

**Comment by madig** [comment]
Next, 

* `dieresiscomb` has a width of 1 in sources/italic/GoogleSansFlexItalic-opsz144-wdth100-wght400-ROND0.ufo and sources/italic/ROND100/GoogleSansFlexItalic-opsz144-wdth100-wght1000-ROND100.ufo but should have a 0.
* `hungarumlautcomb` has a width of 10 in sources/italic/GoogleSansFlexItalic-opsz144-wdth100-wght400-ROND0.ufo but should have 0

**Comment by casasin** [comment]
> Next,
> 
> * `dieresiscomb` has a width of 1 in sources/italic/GoogleSansFlexItalic-opsz144-wdth100-wght400-ROND0.ufo and sources/italic/ROND100/GoogleSansFlexItalic-opsz144-wdth100-wght1000-ROND100.ufo but should have a 0.
> * `hungarumlautcomb` has a width of 10 in sources/italic/GoogleSansFlexItalic-opsz144-wdth100-wght400-ROND0.ufo but should have 0

@joanacorreiatype Could you please check this?

**Comment by joanacorreiatype** [comment]
It's corrected now in all masters.

**Comment by madig** [comment]
@joanacorreiatype Have you pushed your changes to the branch? I don't see it.

**Comment by joanacorreiatype** [comment]
Perdona @madig Ahora ha ido.

**Comment by madig** [comment]
Thanks. The branch builds now: https://github.com/googlefonts/googlesans-flex/actions/runs/5199510447
